tests/integration_tests/performance/test_cpu_template_benchmark.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A benchmark that checks for regression of CPU template operations."""
+
+import json
+import logging
+import os
+import platform
+import shutil
+from pathlib import Path
+
+import pytest
+
+from framework import utils
+from framework.defs import FC_WORKSPACE_DIR
+from host_tools import proc
+
+BENCHMARK_DIRECTORY = "{}/src/vmm".format(FC_WORKSPACE_DIR)
+DEFAULT_BUILD_TARGET = "{}-unknown-linux-musl".format(platform.machine())
+
+PROC_MODEL = proc.proc_type()
+
+NSEC_IN_MSEC = 1000000
+
+BASELINES = {
+    "Intel": {
+        "deserialize": {"target": 0.025, "delta": 0.02}  # milliseconds
```

Comment:
I presume targets are taken from actually measurements.
How were the deltas chosen?

Reply:
Repeated runs to get a band of variance anecdotally. Given the performance margin being sub-millisecond, I think this is adequate for v1.